data_set/advanced/alter.md

```diff
@@ -55,6 +55,10 @@ Alter an existing **data set profile**.
 #### ❌ Raises
 * `SecurityRequestError`<br>
   Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `AlterOperationError`<br>
+  Raises `AlterOperationError` when the **data set profile** cannot be altered because it does not exist in the environment.
```

Comment:
```markdown
* `AlterOperationError`<br>
  Raises `AlterOperationError` when the **data set profile** cannot be altered because it does **NOT** exist.
```

Reply:
Change made in all 39 occurrences of AlterOperationError.